packages/@aws-cdk/aws-eks/lib/managed-nodegroup.ts

```diff
@@ -51,6 +55,22 @@ export interface NodegroupRemoteAccess {
   readonly sourceSecurityGroups?: ISecurityGroup[];
 }
 
+/**
+ * Launch template property specification
+ */
+export interface LaunchTemplateSpecification {
+  /**
+   * The Launch template ID
+   */
+  readonly launchTemplateId: string;
```

Comment:
This should be `ec2.ILaunchTemplate` and not an ID. We don't allow L1s or raw attributes in public APIs of L2s.

Reply:
Hi @eladb 

I created a very basic `ILaunchTemplate` in the managed-nodegroup.ts with `launchTemplateId` as the only property.

Please advise.